todo/constants/health.py

```diff
@@ -0,0 +1,15 @@
+from enum import Enum
+from rest_framework import status
+
+
+class HealthStatus(Enum):
+    UP = status.HTTP_200_OK
+    DEGRADED = status.HTTP_503_SERVICE_UNAVAILABLE
```

Comment:
Are we going to declare these status code for every api?

Reply:
Can you please elaborate?

Reply:
You have defined these two status code for health api.
In future, we will have more apis. Are we going to define it again?

Reply:
I think that will depend on the API. Here status indicates the health state of the server.
If critical components of the server are down then status must be sent as degraded

Reply:
For other apis, we will face the same situation right? So, we will define it again.

Reply:
As discussed this status is particular to the health api